frontend/packages/console-shared/src/utils/utils.ts

```diff
@@ -26,3 +26,15 @@ export const createLookup = <A extends K8sResourceKind>(
   }
   return {};
 };
+
+// Ceph based provisioners
+export const cephStorageProvisioners = [
```

Comment:
This should go in a file specific to storage. utils.ts is too general.

Reply:
Update the ceph plugin to use this value or the helper below. It appears in two other places:

```
frontend/packages/ceph-storage-plugin/src/selectors/index.ts
7:const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];

frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx
7:const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
```